packages/taco/test/conditions/context.test.ts

```diff
@@ -507,115 +508,346 @@ describe('context', () => {
   });
 });
 
-describe('param or context param schema', () => {
-  it('accepts a plain string', () => {
-    expect(paramOrContextParamSchema.safeParse('hello').success).toBe(true);
-  });
+describe.each(['paramOrContextParam', 'rpcParamOrContextParam'])(
+  '%s schema',
+  (schemaName) => {
+    let schema = paramOrContextParamSchema;
+    if (schemaName === 'rpcParamOrContextParam') {
+      schema = rpcParamOrContextParamSchema;
+    }
```

Comment:
Could you have just used the schema directly? i.e.

```suggestion
describe.each([paramOrContextParamSchema, rpcParamOrContextParamSchema])(
  '%s schema',
  (schema) => {
```